docs/page-api.md

```diff
@@ -10,7 +10,7 @@ A binding of URL paths to a Page class that can render the results for those URL
 
 * The HTTP method(s) that are acceptable.
 
-* Default: “get”
+* Default: [“get”, "head"]
```

Comment:
Does react-server even understand `head`, or does it just return the same thing as with `get`? (You'd think I'd know the answer to this...)